generators/server/templates/src/main/java/package/config/_ExcludedFromComponentScan.java

```diff
@@ -0,0 +1,9 @@
+package <%=packageName%>.config;
+
+/**
+ * a simple annotation, to be used for easy exclusion from component scan
+ *
+ * @author David Steiman
+ */
```

Comment:
I'm removing it -> then it's a good idea, but for a separate PR. I'd also like to see if this kind of annotation doesn't already exist, it looks so practical I'm surprised we need to do it.